docs/sending-techniques/sending-data-slack-api-method/sending-data-slack-api-method.md

```diff
@@ -35,7 +35,7 @@ Posting a message with the [`chat.postMessage`](https://docs.slack.dev/reference
     token: ${{ secrets.SLACK_BOT_TOKEN }}
     payload: |
       channel: ${{ secrets.SLACK_CHANNEL_ID }}
-      text: "howdy <!channel>!"
+      text: "howdy <@channel>!"
```

Comment:
@haleychaas I could be wrong, but I think the correct syntax is `<!channel>` 🤔 
https://api.slack.com/reference/surfaces/formatting#special-mentions

I recently fixed this same issue in https://github.com/slackapi/slack-github-action/pull/447 so, I don't think we want to undo that change.